### ISSUE TITLE
[Suggestion] Include a manual for the level editor in the game folder.

### ISSUE THREAD

**Issue by ghost** [alokforever/mh-reinstated#15]
I tried to use the editor to make a level, but I feel lost at what to do once I get pack the first two screens. There seem to be keyboard buttons (like [S] for 'save') that have functions bound to them, but there is no info anywhere which they are.

There is also a lack of a basic tutorial on how to make a level from start to finish.

If there was a tutorial and manual then I could make some levels for the game.

**Comment by zeto0128** [comment]
That would be great. But, unfortunately even I'm not sure how to use that leve editor. Fortunately, some members in the forums (one is Jonax) know how to use the editor. Maybe we can ask him.

**Comment by ghost** [comment]
Hmm, I'll try searching the internet if I can find the editor keyboard shortcuts or a manual made by MHMan in the past. And Jonax too, if he's still around.

In the worst case, I could just go ask MHMan himself about it and see if he has a written document lying around somewhere that explains the use of the editor. I am in contact with him and he's currently working on Overplasma, his new game, so I think it may be possible to get the MH level editor information from him, unless he forgot it too lol.

**Comment by zeto0128** [comment]
Woah can't believe the rumors were true that MHMan is creating a new commercial game. 

As for Jonax, though he may not have a manual, he can help by creating it and share to our small community. After I finish creating around a total of 10 characters or so I'd be contributing some stages too (and use an improved tileset which was given to me by fulkfour).

**Comment by ghost** [comment]
Heck yeah, the trailer to Overplasma looks rad awesome too. I'm happy he's back and that the game has some similarities to multihero, yet being vastly upgraded in all ways. : D

I just finished sending a PM to Jonax23, Denhonator, MHman and Longbow, as they all had done maps previously. They all seemed to be recently active (except MHman, since feb 2018) so there is a good chance I will get a reply from them.

I'll probably compile all the information I get and make guide document that could be included with the game and posted on the forums too. I just hope I'll get enough information to do that.

I'll keep this issue topic open and post progress as it comes.

**Comment by theall** [comment]
Well,you guys still being interested in Multihero, i started the work of miragating this game onto android for education about two months ago, it's not done yet, i will keep working on it if i get any free time, you can access the project from https://github.com/theall/android_examples/tree/master/MultiHero, 
I also created a project to make a Character Editor, a Map Editor and a fully new Engine for this game, sorry that i didn't succeed, the new engine need much work to to, it's very hard to develop them by myself, however i never give up, i hate failure, it's absolutely will be complete in the future.By the way don't try to contact me from twitter, email me if you have any idea and problem.

**Comment by zeto0128** [comment]
Interesting share! I'd be interested in contributing too with those editors and new Engine porting. And long time no see by the way!

**Comment by ghost** [comment]
Neat! Multihero on android sounds excellent, I don't recall there being any smash bros like game that would be available on a smart phone. 

I think it would require using an external plugged in controller to get responsive enough controls though, so I'd recommend adding a recommendation for suitable controllers with the game or its marketing material.

I would also suggest suggest looking into using the either Unity+Playmaker, Godot or Construct 2 engine if you're thinking about creating a multihero successor. Those engines would take care of a lot the difficult and annoying parts of developing the project, while making it easier to focus on creating content.

Meanwhile, I've created this topic over at the tapatalk multihero forums:
https://www.tapatalk.com/groups/multihero/viewtopic.php?f=9&t=60&p=577

I haven't had the opportunity to dive into testing what I've learned so far with the Editor.exe, as I'm currently having a lot of IRL stuff going on, such as moving to a new home, so it may take some time before I finally get the peace and space to get this done.

**Comment by theall** [comment]
You may not know the source of MH is a mess that it's difficult to maintain and extend,I considered using existed Game Engines as you mentioned,and i tried all of them, it was a tough thing to me, finally i thought i need a new engine to avoid the limitation of specified platform like as MH is limited by BlitzBasic.I should show you what i've done:[Demo]https://bitbucket.org/theall/engine/downloads/demo.zip)

Shall i make a gui Editor to let you create new map for MH more easier?This is not hard to me since i ve got a lot of experiences with GUI development.

**Comment by zeto0128** [comment]
With the gui Editor, more people (including me) will be able to create awesome stages to adventure mode to complement the new characters. 

Currently with my 6 (nearly finished characters) I think there would be no major problem porting it since pseudo code is what counts maybe. But for now, I will focus on finishing existing characters before the chance to port to a new engine comes.

**Comment by ghost** [comment]
> 
> 
> You may not know the source of MH is a mess that it's difficult to maintain and extend,I considered using existed Game Engines as you mentioned,and i tried all of them, it was a tough thing to me, finally i thought i need a new engine to avoid the limitation of specified platform like as MH is limited by BlitzBasic.I should show you what i've done:[Demo]https://bitbucket.org/theall/engine/downloads/demo.zip)
> 
> Shall i make a gui Editor to let you create new map for MH more easier?This is not hard to me since i ve got a lot of experiences with GUI development.

Yes, I've seen MHman himself mention its an amateur project multiple times that was coded haphazardly. I'm not suggesting porting the game to another engine, but to learn any of the new engines (I personally recommend Unity + Playmaker, once you learn those via the right video tutorials, it will not feel as hard as you thought it was at first) and then create a new multihero successor game that takes inspiration from the old one, but takes it one step even beyond, this time with a better code organization, better mod support and make it as the MUGEN or modded LittleFighter2 of 2D smash bros gameplay. 

I wouldn't copy multihero itself since it already exists and works. I'd rather see a new evolution of its framework to create a platform for infinite levels, infinite characters, infinite items, infinite adventures and better drop-in-drop-out online cooperative gameplay against AI, both in versus mode gauntlets and adventure mappacks (which perhaps could be generated via procedural generation too?).

Also YES, please do create a gui-powered Editor to create new maps for Multihero. Something like that would make it easier and more likely for people like me and others to actually create large new versus map packs and even new adventure map sequences!
